components/Cards/SchoolCard/__tests__/BackSchoolCard.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import createShallowSnapshotTest from 'test-utils/createShallowSnapshotTest';
+import BackSchoolCard from '../BackSchoolCard';
+
+const locations = [
+  {
+    va_accepted: true,
+    address1: '825 Battery Street',
+    address2: '3rd Floor',
+    city: 'San Francisco',
+    zip: '94111',
+  },
+];
+
+describe('BackSchoolCard', () => {
+  it('should render with required props', () => {
+    createShallowSnapshotTest(
+      <BackSchoolCard cardFlipCallback={jest.fn()} locations={locations} logoSource="logoSource">
+        <p>Testing!</p>
+      </BackSchoolCard>,
+    );
+  });
```

Comment:
Add the following test for me:
```javascript
it('should fire flip callback when close button is pressed', () => {
    const onClickMock = jest.fn();
    const wrapper = mount(
      <BackSchoolCard
        cardFlipCallback={onClickMock}
        locations={locations}
        logoSource="logoSource"
      />,
    );

    wrapper.find('button').simulate('click');
    expect(onClickMock).toHaveBeenCalledTimes(1);
  });
```

This is an integration test ensuring that the CloseButton does what it's supposed to within the BackSchoolCard.